File: fixtures/scale/java/large/src/cryptobench/ciphers/porta/PortaVectors.java
package cryptobench.ciphers.porta;

import java.util.List;

public class PortaVectors {
    public static List<String> samples() {
        return List.of(
            "THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG",
            "PACK MY BOX WITH FIVE DOZEN LIQUOR JUGS",
            "SPHINX OF BLACK QUARTZ JUDGE MY VOW");
    }
}

File: fixtures/scale/scala/large/src/cryptobench/ciphers/porta/PortaVectors.scala
package cryptobench.ciphers.porta

object PortaVectors {
  def samples(): List[String] = List(
      "THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG",
      "PACK MY BOX WITH FIVE DOZEN LIQUOR JUGS",
      "SPHINX OF BLACK QUARTZ JUDGE MY VOW"
  )

  def count(): Int = samples().size
}
